docs_src/explanations/how-scaph-computes-per-process-power-consumption.md

```diff
@@ -1,7 +1,53 @@
 # How scaphandre computes per process power consumption
 
-As you can see with the [prometheus exporter reference](../references/exporter-prometheus.md), scaphandre exporters can provide process level power consumption metrics. This section will explain how it is done and how it may be improved in the future.
+Scaphandre is a tool that makes it possible to see the power being used by a single process on a computer.
+
+This sounds like a simple thing thing to be able to do, but it turns out to be surprisingly complicated. So having a good mental for how it works will make it understand when and how to use Scaphandre.
```

Comment:
I'd suggest to say "it's a bit more subbtle" or any sort of expression like that, rather than "complicated". It seems to me that the complexity is a subjective notion and some people using scaphandre maybe well aware of processes and tasks scheduling.